workflow/rules/qc.smk

```diff
@@ -56,7 +59,7 @@ rule multiqc_lab:
             ]
         ),
         get_fastp_results,
-        get_kraken_output,
+        # get_kraken_output,
```

Comment:
Maybe add a TODO here and also mention the error that is observed (e.g. via a link to the issue on the multiqc repo)?